source/common/quic/quic_filter_manager_connection_impl.h

```diff
@@ -64,6 +64,9 @@ class QuicFilterManagerConnectionImpl : public Network::ConnectionImplBase,
     return network_connection_->connectionSocket()->connectionInfoProvider();
   }
   Network::ConnectionInfoProviderSharedPtr connectionInfoProviderSharedPtr() const override {
+    if (!network_connection_ || !network_connection_->connectionSocket()) {
```

Comment:
Does this break existing tests?

Reply:
I hope not :) If either of those are null when this method is called, we would see a crash in the old code. With the new code, we don't crash, we just return null. CI is green, so I think we're ok.